website/static/js/tests/fangorn.test.js

```diff
@@ -209,21 +209,6 @@ describe('fangorn', () => {
                 item.data.provider = 'github';
                 assert.equal(Fangorn.isInvalidDropItem(folder, item, false, true), true);
             });
-
-            it('cannot be dropped if item provider is figshare and private', () => {
```

Comment:
Adding a test back into getCopyMode to ensure it is copying and not moving would be nice. 

These two tests could be reworded and modified to pass for regression sake.